isacc_messaging/api/isacc_record_creator.py

```diff
@@ -143,9 +152,21 @@ def get_careplan(self, patient_id):
         else:
             return None
 
+    def get_general_practitioner_emails(self, pt: Patient) -> list:
+        emails = []
+        if pt and pt.generalPractitioner:
+            for gp_ref in pt.generalPractitioner:
+                result = HAPI_request('GET', 'Practitioner', gp_ref.id)
+                if result is not None:
+                    gp = Practitioner(result)
```

Comment:
I've just tested at isacc dev, this line is failing with:
```
Exception: Attempting to instantiate <class 'fhirclient.models.practitioner.Practitioner'> with resource data that defines a resourceType of "Bundle"
```
@pbugni  can you please see if it's an easy fix?

```
messagingservice_1  | [2023-05-09 16:25:52,499] ERROR in app: Exception on /sms [POST]
messagingservice_1  | Traceback (most recent call last):
messagingservice_1  |   File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
messagingservice_1  |     response = self.full_dispatch_request()
messagingservice_1  |   File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
messagingservice_1  |     rv = self.handle_user_exception(e)
messagingservice_1  |   File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1821, in handle_user_exception
messagingservice_1  |     reraise(exc_type, exc_value, tb)
messagingservice_1  |   File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
messagingservice_1  |     raise value
messagingservice_1  |   File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
messagingservice_1  |     rv = self.dispatch_request()
messagingservice_1  |   File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
messagingservice_1  |     return self.view_functions[rule.endpoint](**req.view_args)
messagingservice_1  |   File "/opt/app/isacc_messaging/api/views.py", line 109, in incoming_sms
messagingservice_1  |     result = record_creator.on_twilio_message_received(request.values)
messagingservice_1  |   File "/opt/app/isacc_messaging/api/isacc_record_creator.py", line 319, in on_twilio_message_received
messagingservice_1  |     priority=message_priority
messagingservice_1  |   File "/opt/app/isacc_messaging/api/isacc_record_creator.py", line 229, in generate_incoming_message
messagingservice_1  |     notify_emails = self.get_general_practitioner_emails(patient)
messagingservice_1  |   File "/opt/app/isacc_messaging/api/isacc_record_creator.py", line 161, in get_general_practitioner_emails
messagingservice_1  |     gp = Practitioner(result)
messagingservice_1  |   File "/usr/local/lib/python3.7/site-packages/fhirclient/models/practitioner.py", line 70, in __init__
messagingservice_1  |     super(Practitioner, self).__init__(jsondict=jsondict, strict=strict)
messagingservice_1  |   File "/usr/local/lib/python3.7/site-packages/fhirclient/models/domainresource.py", line 42, in __init__
messagingservice_1  |     super(DomainResource, self).__init__(jsondict=jsondict, strict=strict)
messagingservice_1  |   File "/usr/local/lib/python3.7/site-packages/fhirclient/models/resource.py", line 42, in __init__
messagingservice_1  |     super(Resource, self).__init__(jsondict=jsondict, strict=strict)
messagingservice_1  |   File "/usr/local/lib/python3.7/site-packages/fhirclient/models/fhirabstractresource.py", line 23, in __init__
messagingservice_1  |     .format(self.__class__, jsondict['resourceType']))
messagingservice_1  | Exception: Attempting to instantiate <class 'fhirclient.models.practitioner.Practitioner'> with resource data that defines a resourceType of "Bundle"
```